ansible/plugins/callback/datadog_callback.yml

```diff
@@ -0,0 +1 @@
+api_key: "{{DATADOG_API_KEY}}"
```

Comment:
how does this get populated?

Reply:
At present, I am manually doing it. Will use vault in future.